corehq/apps/ota/views.py

```diff
@@ -297,20 +297,23 @@ def heartbeat(request, domain, app_build_id):
         need any validation on it. This is pulled from @uniqueid from profile.xml
     """
     app_id = request.GET.get('app_id', '')
-    build_profile_id = request.GET.get('build_profile_id', '')
+    build_profile_id = request.GET.get('build_profile_id')
 
     info = {"app_id": app_id}
+    # TODO: use get_app_cached? except that's for builds, not apps
```

Comment:
is this still needed? I think this has now moved to the `by_app_id` method on `GlobalAppConfig`

Reply:
yeah, removing this would be good

Reply:
Confirming that we will continue to not use cached version, well because we don't have it?

Reply:
Removed it.